### PR TITLE
Postgres seeding and schema 

### PR DESCRIPTION
**Pull request by jaweesner**
@2000prcs  thanks for the review!

### CODE REVIEW

File: database/postgres/seed.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+//         ok = stream.write(JSON.stringify(dataGen(loop)), 'utf8');
+//       }
+//     } while (loop > 0 && ok);
+//     if (loop > 0) {
+//       stream.once('drain', writer);
+//     }
+//   };
+//   writer();
+// };
+
```
Comment (2000prcs) [inline database/postgres/seed.js]: Let's get rid of unnecessary comments here :)

File: database/postgres/seed.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+const fs = require('fs');
+var i = 1;
+// const projects = [];
+//project row
+const generateOneProjectCSVRow = (id) => {
+  return `${id},` + faker.date.past().toISOString()
+}
+
+const generateOneUpdateCSVRow = (id, foreignLimit) => {
+  return `${id},"${faker.random.words()}",${JSON.stringify(faker.lorem.paragraphs())},${faker.date.past().toISOString()},${faker.random.number()},${Math.ceil(Math.random()*foreignLimit)}`
```
Comment (2000prcs) [inline database/postgres/seed.js]: What does 'foreignLimit' do here?

File: database/postgres/seed.js (hunk 1 above)
Comment (jaweesner) [reply database/postgres/seed.js]: done

File: database/postgres/seed.js (hunk 2 above)
Comment (jaweesner) [reply database/postgres/seed.js]: foreign keys are randomized when creating the data set, the limit prevents creating a ref to a value that doesn't exist

**Pull request merged**